src/oops/SOLID/openClosePrinciple/before/taxes/InternTaxCalculator.java

```diff
@@ -0,0 +1,18 @@
+package oops.SOLID.openClosePrinciple.before.taxes;
+
+import oops.SOLID.openClosePrinciple.before.employees.Employee;
+
+public class InternTaxCalculator implements TaxCalculator {
+
+	private final int INCOME_TAX_PERCENTAGE = 15;
+	private final int MIN_SALARY_FOR_TAX = 300000;
+
+	@Override
+	public double calculate(Employee employee) {
+		if (employee.getMonthlyIncome() > MIN_SALARY_FOR_TAX) {
```

Comment:
I guess the requirement states, annual income

Reply:
Thank you for pointing that out Ankit, will get it fixed.

Reply:
Done!